src/inference/dev_api/memory_solver.hpp

```diff
@@ -81,7 +81,7 @@ class MemorySolver {
 
         // remove unused timestamps (not a begin of some box)
         // each ts should start a box
-        std::vector<bool> ts_exist(max_ts + 1);
+        std::vector<bool> ts_exist((size_t)max_ts + 1);
```

Comment:
What it `max_ts` < 0?

Reply:
Idk why I added these changed to api.
Removed